### PR TITLE
feat: Add Zilla plus schema-registry catalog

### PR DESCRIPTION
**Pull request by vordimous**

### CODE REVIEW

File: src/reference/config/catalogs/catalog-karapace.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 category:
   - Catalog
 ---
 
-# schema-registry Catalog
+# karapace Catalog
 
 ::: important Feature is in Incubator
 Read how to [enable incubator features](../../../how-tos/deploy-operate.md#enable-incubator-features). Star and watch the [Zilla repo](https://github.com/aklivity/zilla/releases) for new releases!
 :::
```
Comment (jfallows) [inline src/reference/config/catalogs/catalog-karapace.md:12]: Please review all features marked incubator as many features moved out of incubator during the last release.

File: src/reference/config/catalogs/catalog-karapace.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
 ---
-shortTitle: schema-registry
-description: Zilla runtime schema-registry catalog
```
Comment (jfallows) [inline src/reference/config/catalogs/catalog-karapace.md:3]: Note: `schema-registry` catalog remains, just in `zilla-plus` now, so `karapace` catalog is net new.

**Review by jfallows — CHANGES_REQUESTED** [review]

File: src/reference/config/catalogs/catalog-karapace.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+### options
+
+> `object`
+
+#### options.url\*
+
+> `string`
+
+Schema Registry URL to access schemas via API calls.
```
Comment (aDaemonThread) [inline src/reference/config/catalogs/catalog-karapace.md:47]: Replace Schema Registry with Karapace Registry

**Review by aDaemonThread — CHANGES_REQUESTED** [review]

**Comment by vordimous** [comment]
@jfallows I think we need to merge this PR until https://github.com/aklivity/zilla-plus/issues/273 is completed and we can make the necessary adjustments in the docs.

**Review by jfallows — APPROVED** [review]

**Pull request merged**